planet/src/main/webapp/js/itinerary.js

```diff
@@ -281,6 +293,40 @@ async function saveEvents() {
     renderPlaceButtons();
 }
 
+function isDateValid(date) {
+    return Date.parse(date) > Date.now();
+}
+
+async function addUserToAllPlaces(userId, eventsSnapshot, date) {
+    // TODO: Consider the real time range of the events
+    let time = 600;
```

Comment:
What does the variable time represent here? Is it the start time of each event? Is this something we want to show on the profile page? If so, would people know the exact time you're going to an event?

Reply:
Actually! No, we are not going to show this info on the profile page but it would be interesting to filter the list of visitors by the time of the visit which is something I doubt we will be able to implement this week but we'll filter by day only.

Reply:
Check the PR's description: the time we are storing the database is not really the time of the visit because the itinerary is yet to be generated.

Reply:
Ah okay!